tests/unit/notice_packager/test_template_generator.py

```diff
@@ -61,6 +63,17 @@ def test_mets2action_mets_xml_generator_with_wrong_action(template_sample_metada
         TemplateGenerator.mets2action_mets_xml_generator(template_sample_metadata)
 
 
+# def test_mets_dmd_rdf_has_html_safe_sequences_after_generation(sample_metadata_with_wrong_title: PackagerMetadata,
```

Comment:
why is this commented? if we don't need delete it

Reply:
Yes, I deleted them.